streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingKeyValueStoreTest.java

```diff
@@ -173,7 +173,7 @@ public void shouldQueryRange() throws Exception {
         store.put(hello, world);
         store.put(hi, there);
         store.put("zooom", "home");
-        final KeyValueIterator<String, String> range = store.range(hello, "zooom");
+        final KeyValueIterator<String, String> range = store.range(hello, hi);
```

Comment:
Because in the old code of `InMemoryKeyValueStore`, our range function is implemented as `subMap(from, true, to, false)` while I changed it to `(from, true, to, true)` in the latest commit.